sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample13_RecognizeInvoicesFromFile.cs

```diff
@@ -42,103 +42,156 @@ public async Task RecognizeInvoicesFromFile()
 
             if (invoice.Fields.TryGetValue("VendorName", out FormField vendorNameField))
             {
-                if (vendorNameField.Value.ValueType == FieldValueType.String)
+                string vendorName = vendorNameField.Value.AsString();
```

Comment:
Why would this not be AsStringOrNull() like the others?  Is that because null is a valid option for String?

Reply:
the service doesn't return a `ValueType` when the normalization to a type fails. After I confirmed with the service, this will only happen for types `number, integer, date, time, selectionMark, country, phoneNumber, and gender`.
I added `xxOrNull` for those types.